### PR TITLE
Support for Dropdown Rendering Type

### PR DESCRIPTION
**Pull request by ShekharGupta1988**
Adding Support for Dropdown View. The View Model is a (key: string, value: Response) pair where:
- "key" goes in the dropdown list
- "value" is the Response (insights, graphs, markdown etc.) that gets displayed when a key is selected.

### CODE REVIEW

**Comment by ShekharGupta1988** [comment]
Adding Unit Tests

File: src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        /// First Item represents item in Dropdown, Second Item represents whether selected by default in dropdwon, Third Item represents the body after dropdown selection
+        /// </summary>
+        public List<Tuple<string, bool, Response>> Data;
+
+        /// <summary>
+        /// Creates an instance of Dropdown Class
+        /// </summary>
+        /// <param name="label">Dropdown Label</param>
+        /// <param name="data">Dropdown Data</param>
+        public Dropdown(string label, List<Tuple<string, bool, Response>> data)
```
Comment (stevenernst131) [inline src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs]: What would happen if multiple are selected? Might be a question for the UI

**Comment by stevenernst131** [comment]
This looks good, you may want to do UI changes first, so that you don't have to go back and make updates here.

File: src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs (hunk 1 above)
Comment (ShekharGupta1988) [reply src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/Dropdown.cs]: the last one gets selected :)

**Comment by ShekharGupta1988** [comment]
Ya, I have done UI changes also and validated end-to-end.

**Comment by stevenernst131** [comment]
Awesome!

**Pull request merged**